### PR TITLE
USERGUIDE: added new userguide.md and cleaned up readme.md

### PR DESCRIPTION
**Pull request by mofosyne**
This is in response to https://github.com/fablabnbg/inkscape-silhouette/issues/261

### CODE REVIEW

File: USERGUIDE.md [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+
+### Adding Print and Cut Layers
+
+You are recommended to add a layer named `Print` and `Cut`.
+The `Print` layer in addition to `Regmarks` will be ignored by `Send to Silhouette` extention.
+
+
+### Add registration mark
+
+<img src="https://github.com/mofosyne/inkscape-silhouette/assets/827793/c9c32197-4967-42f9-b8fa-f407d4f12203" alt="screenshot of the regmarked document" height=200px>
```
Comment (t0b3) [inline USERGUIDE.md]: can you add the screenshot files to the repo ?

File: USERGUIDE.md [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+
+
+---
+
+## Plot
+
+<img src="https://github.com/mofosyne/inkscape-silhouette/assets/827793/859b134f-8765-4768-89c0-a5a9fe569243" alt="screenshot of Send to Silhouette" height=200px>
+
+1. Open your document with inkscape.
+2. Ensure the unit of document width and height is mm or inch, but not px. (File - Document settings - Page - Custom - Unit mm) Otherwise you may observe differences in dimensions at inkscape 0.91/0.92, because default dpi has changed from 90 to 96.
```
Comment (t0b3) [inline USERGUIDE.md]: drop this line in favor of a note: 
- note: documents in px are plotted at 96dpi.

File: USERGUIDE.md [hunk 3]
```diff
@@ -0,0 +1,112 @@
...
+3. Convert all objects and texts to paths (Path - Convert object to path)
```
Comment (t0b3) [inline USERGUIDE.md]: update this line:
3. Convert text objects to paths (Path - Convert object to path)

File: USERGUIDE.md [hunk 4]
```diff
@@ -0,0 +1,112 @@
...
+
+1. Open the document which fit to your setup (e.g. With rendered registration mark as illustrated above)
+2. Insert your cutting paths and graphics on the apropriate layers.
+3. Printout the whole document including registration marks. You probably want to hide the cutting layer.
+4. Select your cutting paths in the document, but exclude regmarks and graphics.
+    - The extention is smart enough to ignore any layers that has the word `Print` or `Regmarks` in it.
+5. On the **Regmarks** tab:
+  * Check **Document has registration marks**
+  * Check **Search for registration marks**
+6. Set all following parameters according to the registration file used:
```
Comment (t0b3) [inline USERGUIDE.md:72]: add a note: values are read from regmarks layer if 0 is entered

**Review by mofosyne — COMMENTED** [review]
Corrected. Also move the images to repo

File: USERGUIDE.md [hunk 5]
```diff
@@ -0,0 +1,113 @@
...
+
+
+---
+
+## Plot
+
+<img src="./assets/screenshot_of_send_to_silhouette.png" alt="screenshot of Send to Silhouette" height=200px>
+
+1. Open your document with inkscape.
+2. Ensure the unit of document width and height is mm or inch, but not px. (File - Document settings - Page - Custom - Unit mm)
```
Comment (t0b3) [inline USERGUIDE.md]: this line is no longer needed - you may drop it and rearrange the numbering...
2. Ensure the unit of document width and height is mm or inch, but not px. (File - Document settings - Page - Custom - Unit mm)

**Comment by mofosyne** [comment]
Okay removed and list number updated

**Pull request merged**